.github/workflows/buildtest.yml

```
@@ -167,12 +183,22 @@ jobs:
           vim_version: '${{ matrix.vim }}'
           vim_type: '${{ matrix.type }}'
           download: '${{ matrix.vim_download || matrix.download }}'
-      - name: Build
+      - name: Setup vim-themis
+        uses: actions/checkout@v2
+        with:
+          repository: thinca/vim-themis
+          ref: ${{ env.THEMIS_VERSION }}
+          path: ${{ github.workspace }}/vim-themis
+      - name: Setup vimproc
+        uses: actions/checkout@v2
+        with:
+          repository: Shougo/vimproc.vim
+          ref: ${{ env.VIMPROC_VERSION }}
+          path: ${{ github.workspace }}/vimproc
+      - name: Fetch vimproc dll
         shell: pwsh
         run: |
-          git -c advice.detachedHead=false clone https://github.com/thinca/vim-themis  --quiet --branch ${Env:THEMIS_VERSION} --single-branch --depth 1 ${Env:GITHUB_WORKSPACE}\vim-themis
-          git -c advice.detachedHead=false clone https://github.com/Shougo/vimproc.vim --quiet --branch ver.9.2               --single-branch --depth 1 ${Env:GITHUB_WORKSPACE}\vimproc
-          Invoke-WebRequest -Uri "https://github.com/Shougo/vimproc.vim/releases/download/ver.9.2/vimproc_win64.dll" -OutFile "${Env:GITHUB_WORKSPACE}\vimproc\lib\vimproc_win64.dll"
+          Invoke-WebRequest -Uri "https://github.com/Shougo/vimproc.vim/releases/download/${Env:VIMPROC_VERSION}/vimproc_win64.dll" -OutFile "${Env:GITHUB_WORKSPACE}\vimproc\lib\vimproc_win64.dll"
```

こちらを

```suggestion
          Invoke-WebRequest -Uri "https://github.com/Shougo/vimproc.vim/releases/download/ver.${Env:VIMPROC_VERSION}/vimproc_win64.dll" -OutFile "${Env:GITHUB_WORKSPACE}\vimproc\lib\vimproc_win64.dll"
```

としたほうがいいかなと思いました。